_topic_map.yml

```diff
@@ -511,6 +511,9 @@ Topics:
   - Name: Creating policy for Operator installations and upgrades
     File: olm-creating-policy
     Distros: openshift-origin,openshift-enterprise
+  - Name: Configuring OLM in disconnected mode
```

Comment:
The release mirroring PRs went with "restricted network" instead of "disconnected" (e.g. see #16696).